uPortal-spring/src/main/java/org/apereo/portal/spring/security/preauth/PortalPreAuthenticatedProcessingFilter.java

```diff
@@ -54,7 +54,15 @@
 public class PortalPreAuthenticatedProcessingFilter
         extends AbstractPreAuthenticatedProcessingFilter {
 
-    private final Log swapperLog = LogFactory.getLog("org.jasig.portal.portlets.swapper");
+    private static final String SWAPPER_LOG_NAME = "org.jasig.portal.portlets.swapper";
+
+    /** Log for identity swapper activity. */
+    private final Logger swapperLog = LoggerFactory.getLogger(SWAPPER_LOG_NAME);
+
+    /**
+     * "Regular" log. This variable covers a commons-logging log of the same name in the superclass.
+     */
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Since `PortalPreAuthenticatedProcessingFilter` is non-final, should this logger be `protected` and so available to potential sub-classes?

Reply:
I usually prefer it when only methods are `protected`, not instance members at all.

Reply:
Also, I'd say that `PortalPreAuthenticatedProcessingFilter` is non-final for the sake of not interfering with Spring proxy magic... not b/c there's any intent to support a human subclassing it.